addon/components/lazy-render-wrapper.js

```diff
@@ -102,7 +103,7 @@ export default Ember.Component.extend({
 
 			return true;
 
-		} else if (this.get('isShown') || this.get('tooltipIsVisible')) {
+		} else if (!this.get('disabled') && (this.get('isShown') || this.get('tooltipIsVisible'))) {
 
 			this.set('_hasRendered', true);
 			return true;
```

Comment:
And what about in 128, if i skip the parent event stuff, and then disabled gets set to false, would that be rerun anywhere else?